components/brave_shields/common/features.cc

```diff
@@ -10,6 +10,8 @@
 namespace brave_shields {
 namespace features {
 
+const base::Feature kBraveAdblockCnameUncloaking{
+    "BraveAdblockCnameUncloaking", base::FEATURE_ENABLED_BY_DEFAULT};
```

Comment:
given that it leaks, we may want to disable it on MacOS?

Reply:
also would be nice to extend tests with corresponding checks

Reply:
++ we should definitely make sure tests cover enabled/disabled